index.js

```diff
@@ -0,0 +1,957 @@
+const ONESIGNAL_SDK_ID = 'onesignal-sdk';
```

Comment:
Another point to consider is that from what I've seen, we're now using `.js` files instead of `.ts` files directly.

This is probably not a good choice since we're not having any benefit from TypeScript, and we're simply injecting a type file into the package to determine the types.

Instead, we could use typescript in all files (`.ts`) and have the benefits of typescript directly in our project. We could then use the typescript compiler to generate the final package.

Reply:
that's a good observation, but to make this more easily maintainable we are transitioning to a code-generated version of this project. the JS in this repository was already transpiled from the TS in the code-generator

Reply:
Awesome, I did not consider this. I'm marking this as resolved then :)